core/modules/dashboard/dashboard.module

```diff
@@ -0,0 +1,356 @@
+<?php
+/**
+ * @file
+ * Provides an administrative dashboard.
+ */
+
+
+/**
+ * Implements hook_menu().
+ */
+function dashboard_menu() {
+  $items = array();
+  $dashboard = layout_load('dashboard');
+  if ($dashboard && !$dashboard->disabled) {
+    $items['admin/dashboard/settings'] = array(
+      'title' => 'Settings',
+      'page callback' => 'backdrop_get_form',
+      'page arguments' => array('dashboard_admin_settings'),
+      'description' => 'Admin dashboard settings',
+      'access arguments' => array('access content'),
```

Comment:
Seems this should have a different permission? Do we want/need "Administer dashboard" permission?

Reply:
+1 for "administer dashboard" permission.